composer.json

```diff
@@ -10,37 +10,41 @@
     ],
     "require": {
         "php": "^7",
-        "symfony/symfony": "^3.4",
-        "twig/twig": "^2",
+        "symfony/console": "^4.4",
+        "twig/twig": "^2 || ^3",
         "zicht/util": "~1.1"
```

Comment:
Ik mis hier de zicht admin bundle, doctrine, sf framework, sf form, gedmo tree, etc.

Ik snap ook dat je niet altijd alles gebruikt uit de Zicht framework extra bundle, zoals forms of tree builder. Misschien dan toevoegen in de `suggest` sectie.

Reply:
Ik heb hier nu 1 SF dep gekozen om de versie te kunnen afdwingen, op zich is dit voor de upgrade afdoende.

Reply:
Ik heb toegevoegd:
```
        "symfony/form": "^4.4",
        "symfony/http-foundation": "^4.4",
        "symfony/routing": "^4.4"
```
Deze had ik minimaal nodig om de unit tests succesvol te kunnen draaien.